### PR TITLE
Seth/remove assistant beta label

### PR DESCRIPTION
**Pull request by jaksiks**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

# !!! Please don't merge until the GA Launch !!!

### CODE REVIEW

File: docs/semgrep-code/semgrep-assistant-code.md [hunk 1]
```diff
@@ -26,9 +26,8 @@ This document provides an overview of the following:
 * How to enable Semgrep Assistant
 * How to receive notifications or comments from Semgrep Assistant
 
-:::info Feature maturity 
-* This feature is in **public beta**.
-* You or your developers may encounter rough edges. This includes inaccurate recommendations or broken text. Please leave feedback on the PR comments from Semgrep Assistant to help the Semgrep team improve Semgrep Assistant.
+:::info Assistant content 
+Assistant content is generated by LLMs. Please review carefully and leave feedback on the PR comments from Semgrep Assistant to help the Semgrep team improve Semgrep Assistant.
```
Comment (jaksiks) [inline docs/semgrep-code/semgrep-assistant-code.md:30]: Not sure if keeping a little blurb in there would be helpful or not. Open to suggestions / deleting this

Comment (khorne3) [reply docs/semgrep-code/semgrep-assistant-code.md:30]: I think the only thing I would change is add what LLM stands for

Comment (s-santillan) [inline docs/semgrep-code/semgrep-assistant-code.md:30]:
```suggestion
Assistant content is generated by large language models (LLMs). Please review carefully and leave feedback on the PR comments from Semgrep Assistant to help the Semgrep team improve Semgrep Assistant.
```

**Review by s-santillan — APPROVED** [review]
Added 1 suggestion based on Katie's note.

I s'pose we hold off merging this until release date?

**Review by khorne3 — APPROVED** [review]

File: docs/semgrep-code/semgrep-assistant-code.md (hunk 1 above)
Comment (jaksiks) [reply docs/semgrep-code/semgrep-assistant-code.md]: resolved below, thanks Kaite!

**Pull request merged**